src/cloudWatchLogs/commands/searchLogGroup.ts

```diff
@@ -202,9 +198,9 @@ export class SearchLogGroupWizard extends Wizard<SearchLogGroupWizardResponse> {
         })
 
         this.form.submenuResponse.bindPrompter(createRegionSubmenu)
+        this.form.timeRange.bindPrompter(() => new TimeFilterSubmenu())
```

Comment:
This changes the prompt order which the tests check.

Reply:
that's intentional. i have some more changes to push, but the idea was that if the service rejects the search input, the wizard should not be hidden, the user should be able to edit the search input.